cmd/auth-rest/startcmd/start.go

```diff
@@ -242,6 +245,12 @@ func startAuthService(parameters *authRestParameters, srv server) error {
 		router.HandleFunc(handler.Path(), handler.Handle()).Methods(handler.Method())
 	}
 
+	// TODO #34 add the handlers from this controller to the router so that the endpoints are reachable
+	_, err = rp.New(&operation.Config{Provider: provider})
```

Comment:
Change rp to auth as rp was for relying party

Reply:
done